pkg/lockfile/parse-npm-lock.go

```diff
@@ -65,6 +68,13 @@ func parseNpmLockDependencies(dependencies map[string]NpmLockDependency) map[str
 		finalVersion := version
 		commit := ""
 
+		// If the package is aliased, get the name and version
+		if strings.HasPrefix(detail.Version, "npm:") {
+			i := strings.Index(detail.Version, "@")
```

Comment:
This will need to use `strings.LastIndex` to handle packages with scopes e.g. `npm:@foo/bar@1.2.3`

Reply:
good point! I have also added a test case for this.